Makefile

```diff
@@ -23,7 +23,13 @@ run-mock-registry: ${GOPATH}/bin/mock-registry vendor
 prepare-build: install
 	cp "${GOPATH}"/bin/broker build/broker
 
-build: prepare-build
+dev-image: prepare-build
+	docker build -f ${BUILD_DIR}/Dockerfile-src ${BUILD_DIR} -t ${BROKER_IMAGE}:${TAG}
+	@echo
+	@echo "Remember you need to push your image before calling make deploy"
+	@echo "    docker push ${BROKER_IMAGE}:${TAG}"
+
+release-image:
```

Comment:
Perhaps it should be ```make binary-image```? or ```make build-binary```? I look at release as a tagged release of code. So it's something already tested and stable that will be published with the appropriate tag. I don't view a release as something that's currently under heavy development. I'm not saying that's what you're suggestion, but it's further clarification of my initial comment.